yoga/Yoga.c

```diff
@@ -941,6 +955,41 @@ static inline YGDirection YGNodeResolveDirection(const YGNodeRef node,
   }
 }
 
+static float YGBaseline(const YGNodeRef node,
+                        const YGFlexDirection crossAxis) {
+  if (node->baseline != NULL) {
+    const float baseline = node->baseline(node);
+    if (YGFloatIsUndefined(baseline)) {
+      return node->layout.measuredDimensions[dim[crossAxis]];
```

Comment:
This will always be height as baseline is undefined for column flex direction. right? in that case I think it is better to be clear and use `YGDimensionHeight` instead of `dim[crossAxis]`